### ISSUE TITLE
Missing focus trap on overlays and dropdowns

### ISSUE THREAD

**Issue by marcfilleul** [htmlstreamofficial/preline#128]
Hi,

Usually we want to have some focus trap inside the overlays or dropdowns content for accessibility purpose.
When opening this kind of content, we shouldn't be able to access outside links with TAB.

Would it be possible to add this behaviour by default?
With eventually the option to disable the focus trap if needed (even if it's not recommended).

Thanks.

**Comment by jahaganiev** [comment]
Hey @marcfilleul - thanks a lot for the suggestion. We've addressed this in our latest release, please check out here for the demo https://preline.co/docs/modal.html

Looking forward to any further feedback you might have.

Cheers!
